core/contracts/oracle/implementation/EncryptedStore.sol

```diff
@@ -0,0 +1,50 @@
+pragma solidity ^0.5.0;
+
+/**
+ * @title Contract for storing keyed encrypted messages via the EVM.
+ * @dev This contract uses topic hashes as keys and can store a single arbitrary encrypted message per topic at any
+ * given time.
+ * @dev There's technically nothing that requires the topics hashed or for the messages to be encrypted.
```

Comment:
I think our current convention is to use only 1 `@dev` natspec flag.

Reply:
done